addon/components/pix-banner.js

```diff
@@ -1,6 +1,24 @@
 import Component from '@glimmer/component';
 
 export default class PixBanner extends Component {
+  get icon() {
+    let icon = 'info-circle';
+    if(this.args.type === 'error') {
+      icon = 'exclamation-triangle';
+    } else if (this.args.type == 'warning') {
+      icon = 'exclamation-circle';
+    }
+    return icon;
+  }
+
+  get className() {
+    let className ='information';
+    if(this.args.type === 'error' || this.args.type === 'warning') {
+      className = this.args.type;
+    }
+    return className;
+  }
```

Comment:
Une autre solution pour lire plus simplement que l'enchaînement des `if` serait d'utiliser une structure de ce genre : 
```
const TYPE_INFO = 'information';
const TYPE_ERROR = 'error';
const TYPE_WARNING = 'warning';
const types = [TYPE_INFO, TYPE_ERROR, TYPE_WARNING];
const icons = {
  [TYPE_INFO]: 'info-circle',
  [TYPE_ERROR]: 'exclamation-triangle',
  [TYPE_WARNING]: 'exclamation-circle',
};
```

Le `get icon` et le `className` pourraient donc s'écrirent comme ceci : 
```suggestion
get icon() {
    return icons[this.type];
}
  
get type() {
   return types.includes(this.args.type) ? this.args.type : TYPE_INFO;
}
```
Comme ça en plus on évite de parler de `className` dans le `js`.

Reply:
👍